src/transforms/replace.jl

```diff
@@ -0,0 +1,29 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+"""
+    Replace(old₁ => new₁, old₂ => new₂, ..., oldₙ => newₙ)
+
+Replaces all occurrences of `oldᵢ` with `newᵢ` in the table.
+"""
+struct Replace{K,V} <: Colwise
+  oldnew::IdDict{K,V}
```

Comment:
Should we call this pairs for consistency with other transforms? Also, first time I see IdDict, interesting finding.

Reply:
OK.
IdDict is used for this reason:
```julia-repl
julia> Dict((1 => 3, true => false))
Dict{Integer, Integer} with 1 entry:
  true => false

julia> IdDict((1 => 3, true => false))
IdDict{Integer, Integer} with 2 entries:
  1    => 3
  true => false
```